bin/nlg-eval

```diff
@@ -58,27 +63,37 @@ def _download_file(d):
 
 
 def setup(ctx, param, value):
+    from nltk.downloader import download
+    download('punkt')
+
     if not value:
         return
     from multiprocessing import Pool
 
-    data_path = os.getenv('NLGEVAL_DATA', 'nlgeval/data')
-    path = 'nlgeval/word2vec/glove2word2vec.py'
+    if value == 'default':
+        data_path = os.getenv('NLGEVAL_DATA', os.path.join(XDG_CACHE_HOME, 'nlgeval'))
+    else:
+        data_path = value
+    click.secho("Installing to {}".format(data_path), fg='red')
+    click.secho("In case of incomplete downloads, delete the directory and run `nlg-eval --setup {}' again.".format(data_path),
+               fg='red')
+
+    path = os.path.join(CODE_PATH, 'nlgeval/word2vec/glove2word2vec.py')
     if os.path.exists(path):
-        os.remove('nlgeval/word2vec/glove2word2vec.py')
+        os.remove(path)
 
     downloads = []
 
     if sys.version_info[0] == 2:
         downloads.append(dict(
             url='https://raw.githubusercontent.com/manasRK/glove-gensim/42ce46f00e83d3afa028fb6bf17ed3c90ca65fcc/glove2word2vec.py',
-            target_dir='nlgeval/word2vec'
+            target_dir=os.path.join(CODE_PATH, 'nlgeval/word2vec')
         ))
     else:
         # Change URL once https://github.com/robmsmt/glove-gensim/pull/1 has been merged.
         downloads.append(dict(
             url='https://raw.githubusercontent.com/juharris/glove-gensim/4c2224bccd61627b76c50a5e1d6afd1c82699d22/glove2word2vec.py',
```

Comment:
They merged the PR! While you're changing stuff would you change this URL to `https://raw.githubusercontent.com/robmsmt/glove-gensim/4c2224bccd61627b76c50a5e1d6afd1c82699d22/glove2word2vec.py`?

Reply:
done